File: src/mappings/plugin_manager.rs
pub fn get(filetype: &str) -> Option<(&str, &str)> {
    let plugin_manager = match filetype {
        "lazy" => ("default", "Lazy"),
        "pckr" => ("default", "Pckr"),
        "packer" => ("default", "Packer"),
        _ => return None,
    };

    Some(plugin_manager)
}

File: lua/cord/mappings/plugin_manager.lua
local M = {}

local mappings = {
  lazy = { 'default', 'Lazy' },
  pckr = { 'default', 'Pckr' },
  packer = { 'default', 'Packer' },
}

M.get = function(filetype) return mappings[filetype] end

return M
